packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -1,6 +1,7 @@
 import fs from 'fs/promises';
 import path from 'path';
 import chalk from 'chalk';
+import got from 'got';
```

Comment:
Ack, I feel so bad saying this, but since this package already has an HTTP request library (`node-fetch`), we _probably_ should use that one instead of adding a new one. Having two different HTTP request libraries in the same project feels a bit redundant and might be confusing for folks adding code later.  Are you okay switching over to `node-fetch`? It has a [good API](https://www.npmjs.com/package/node-fetch)! (It's just the fetch API!)

Reply:
Oh bummer...I'll rework this in `node-fetch`.

Reply:
Fixed in c572c75b88c3213cf885b21230bdf1d732f0da23.